src/Minsk/CodeAnalysis/Emit/Emitter.cs

```diff
@@ -248,7 +248,7 @@ private void EmitStatement(ILProcessor ilProcessor, BoundStatement node)
             switch (node.Kind)
             {
                 case BoundNodeKind.NopStatement:
-                    EmitNopStatement(ilProcessor, (BoundNopStatement)node);
+                    EmitNopStatement(ilProcessor);
```

Comment:
I'm seeing a tradeoff here:

- Consistency over all Emit\* methods; vs
- Removing unused node from **some** of the Emit\* methods.

Not sure which way you prefer.